docs/general/rendering.md

```diff
@@ -144,6 +144,32 @@ That's the result:
 
 This feature is specific to JsonApi, so you have to use the use the [JsonApi Adapter](adapters.md#jsonapi)
 
+
+##### Resource-level
+
+In your serializer, define each link in one of the following methods:
+
+As a static string
+
+```ruby
+link :link_name, 'https://example.com/resource'
+```
+
+As a block to be evaluated. When using Rails, URL helpers are available.
```

Comment:
Tests w grape adapter?

B mobile phone

> On Mar 3, 2016, at 12:14 PM, Ben Mills <notifications@github.com> wrote:
> 
> 
> Since having revised this PR to use SerializationContext I think we need:
> 
>  Docs for configuring Rails.application.routes.default_url_options
>  Docs that give an example of url_for. (as previously mentioned)
> Anything else?
> 
> —
> Reply to this email directly or view it on GitHub.
>